packages/hawtio/src/plugins/connect/jolokia-service.ts

```diff
@@ -500,20 +503,23 @@ class DummyJolokia implements IJolokia {
     opts?: ISimpleOptions,
   ) {
     /* no-op */
+    opts?.success?.(unknown)
```

Comment:
it's not "no-op" anymore :-)

Also I don't know if it's the correct usage of superstruct `unknown` here. Can't you rather simply do `opts?.success?.({})`?